js/api/Forms.js

```diff
@@ -37,7 +37,27 @@ export function loadCachedFormDataById(formId) {
 
 // Fetches the cached forms related to a specific survey
 export function loadCachedForms(surveyId) {
-  return realm.objects('Form').filtered(`surveyId = "${surveyId}"`)
+  let forms = realm.objects('Form').filtered(`surveyId = "${surveyId}"`)
+  let submissions = realm.objects('Submission')
+  let formList = []
+
+  // Switch out from Realm list to standard array
```

Comment:
https://realm.io/docs/react-native/latest/api/Realm.Collection.html

I think its an instance of Realm.Collection, which has an .entries() method as the iterable interface